src/voices/voices.rs

```diff
@@ -16,6 +15,13 @@ impl<T> WithCreationTime<T> {
             creation_time: SystemTime::now(),
         }
     }
+
+    fn new_with_time(item: T, creation_time: SystemTime) -> Self {
```

Comment:
At one point I wanted to specify time in my tests, then I realised input order probably suffices.